### ISSUE TITLE
``sphinxcontrib-jsmath`` -- PyPI

### ISSUE THREAD

**Issue by AA-Turner** [sphinx-doc/sphinx#11559]
@tk0miya -- please would you be able to add me as an owner to ``sphinxcontrib-jsmath`` [on PyPI](https://pypi.org/project/sphinxcontrib-jsmath/)? I'm currently unable to upload a new release (see e.g. #11495, I've made the packaging changes in the repo but they're not yet released).

Thanks,
Adam

**Comment by terencehonles** [comment]
@tk0miya friendly reminder, I noticed that `sphinxcontrib-jsmath` is quite out of date and the unreleased changes fix some deprecation warnings I have. I was going to open a ticket since the issue tracker there is not available, but I found this before doing so.

Thanks in advance!

**Comment by sallner** [comment]
There is also an email address of @tk0miya on the github profile, which he might read if not only a notification is sent. I am also stuck with the implicit namespace issue in several projects. Would love to see some progress on that. Thanks for your work @AA-Turner .

**Comment by djhoese** [comment]
Good point @sallner on the email. @AA-Turner let us know if you don't hear anything. I figure you should probably email them since you're the one who needs permissions.

**Comment by AA-Turner** [comment]
I have created a PyPI support request: https://github.com/pypi/support/issues/7118.

A

**Comment by AA-Turner** [comment]
@tk0miya kindly got in touch and sent an invitation, the PyPI project is now transferred to the Sphinx organisation!

A
